docs/src/main/sphinx/sql/select.md

```diff
@@ -67,6 +68,27 @@ ROLLUP ( column [, ...] )
 
 Retrieve rows from zero or more tables.
 
+## WITH FUNCTION clause
+
+The `WITH FUNCTION` clause allows you to define a list of inline SQL routines
+that are available for use in the rest of the query.
+
+The following example declare and uses two inline routines:
+
+```sql
+WITH FUNCTION hello(name varchar)
```

Comment:
Should we format like
```
WITH
  FUNCTION hello(name varchar)
    RETURNS varchar
    RETURN format('Hello %s!', 'name'),
  FUNCTION bye(name varchar)
    RETURNS varchar
    RETURN format('Bye %s!', 'name')
```
That matches how we format `WITH` below

Reply:
Yes.. done